internal/tofu/node_resource_import.go

```diff
@@ -251,6 +265,14 @@ func (n *graphNodeImportStateSub) Execute(ctx EvalContext, op walkOperation) (di
 		return diags
 	}
 
+	// Insert marks from configuration
+	if n.Config != nil {
+		// Since the import command allow import resource with incomplete configuration, we ignore diagnostics here
```

Comment:
Shouldn't this technically be the same for plannable import as well? We might need to ignore diagnostics there as well

Reply:
In fact, before plannable import, the validate walk will perform this evaluate to make sure the config is valid, so it will never fail when plannable import. 
Ignore or not are both acceptable, but I prefer to check it to make the code looks clean. 

https://github.com/opentofu/opentofu/blob/739a66a86c4d775adf60d67cdc6c066059e46c12/internal/tofu/node_resource_validate.go#L383